### PR TITLE
Manage the styling of banner of both task update and standup update

### PR DESCRIPTION
**Pull request by Pratiyushkumar**
### Description:
 Manage the styling of the banner of both task update and standup update

### Dev Tested:
- [x] Yes

### Images/video of the change:
image
image

### CODE REVIEW

**Review by RitikJaiswal75 — DISMISSED** [review]

**Review by shubhamsigdar1 — APPROVED** [review]

**Review by shubhamsigdar1 — APPROVED** [review]

**Review by sahsisunny — DISMISSED** [review]
Please add test with coverage screenshot

**Review by RitikJaiswal75 — APPROVED** [review]

**Comment by Pratiyushkumar** [comment]
> Please add test with coverage screenshot

I have just worked on styling and for styling there are no test coverage

**Comment by sahsisunny** [comment]
Why did you dismiss my REQUEST CHANGES? Without any proper reason don't delete or dismiss any review or comments please.

**Review by sahsisunny — APPROVED** [review]

**Comment by sahsisunny** [comment]
> > Please add test with coverage screenshot
> 
> I have just worked on styling and for styling there are no test coverage

You have made some changes in  .tsx files so I think a test is needed.

File: src/components/ProgressForm/ProgressLayout.tsx [hunk 1]
```diff
@@ -20,12 +20,10 @@ const ProgressLayout: FC = () => {
...
         <>
             <NavBar />
-            <div className={styles.banner}>
-                <ProgressHeader
-                    totalMissedUpdates={totalMissedUpdates}
-                    updateType="Progress"
-                />
-            </div>
+            <ProgressHeader
+                totalMissedUpdates={totalMissedUpdates}
```
Comment (harshith-venkatesh) [inline src/components/ProgressForm/ProgressLayout.tsx:24]: Are we handling isNaN scenarios here?

Comment (Pratiyushkumar) [reply src/components/ProgressForm/ProgressLayout.tsx:24]: yes, we are handling this as if the date array length is 0 so it should so 0, and inside the array we have restricted array element type to be numbers. so I guess NaN situation won't occur

**Review by harshith-venkatesh — APPROVED** [review]
Good work handling all scenarios and explaining them

**Pull request merged**